src/app/experience/experience.component.css

```diff
@@ -6,3 +6,11 @@
   font-size: 12pt;
   color: darkgray;
 }
+
+th.looking {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the name of this overqualified element. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=css%3Aoverqualified-elements)